nym-wallet/src-tauri/src/operations/app/link.rs

```diff
@@ -0,0 +1,22 @@
+use std::process::Command;
+
+#[tauri::command]
+pub async fn open_url(url: String) -> Result<(), String> {
+    println!("Opening URL: {}", url);
+
+    let status = match std::env::consts::OS {
+        "macos" => Command::new("open").arg(&url).status(),
+        "windows" => Command::new("cmd").args(["/c", "start", &url]).status(),
+        "linux" => Command::new("xdg-open").arg(&url).status(),
+        os => return Err(format!("Unsupported OS: {}", os)),
+    };
```

Comment:
yea instead of this manual implem you should use the opener plugin https://tauri.app/plugin/opener/

Reply:
Now using the plugin